.github/workflows/maven-compiler.yml

```diff
@@ -13,6 +13,7 @@ on:
     paths:
     - 'src/**'
     - 'pom.xml'
+    - '.github/workflows/**'
```

Comment:
I mean... I think that only this particular action is necessary here, not all actions...